src/confd/yang/confd.inc

```diff
@@ -20,24 +20,24 @@ MODULES=(
 	# NOTE: ietf-tls-client must be version matched with ietf-tls-server, used by netopeer2!
 	"ietf-tls-client@2023-12-28.yang" # used by ietf-syslog (remote)
 	"ietf-syslog@2024-03-21.yang -e file-action -e file-limit-size -e remote-action"
-	"infix-syslog@2024-07-19.yang"
+	"infix/infix-syslog@2024-07-19.yang"
 	"iana-hardware@2018-03-13.yang"
 	"ietf-hardware@2018-03-13.yang -e hardware-state"
-	"infix-hardware@2024-04-25.yang"
+	"infix/infix-hardware@2024-04-25.yang"
 	"ieee802-dot1q-types@2022-10-29.yang"
-	"infix-ip@2024-09-16.yang"
-	"infix-if-type@2024-01-29.yang"
-	"infix-routing@2024-09-23.yang"
+	"infix/infix-ip@2024-09-16.yang"
+	"infix/infix-if-type@2024-01-29.yang"
+	"infix/infix-routing@2024-10-01.yang"
 	"ieee802-dot1ab-lldp@2022-03-15.yang"
-	"infix-lldp@2023-08-23.yang"
-	"infix-dhcp-client@2024-09-20.yang"
-	"infix-meta@2024-06-19.yang"
-	"infix-system@2024-09-13.yang"
-	"infix-services@2024-05-30.yang"
+	"infix/infix-lldp@2023-08-23.yang"
+	"infix/infix-dhcp-client@2024-09-20.yang"
+	"infix/infix-meta@2024-06-19.yang"
+	"infix/infix-system@2024-09-13.yang"
+	"infix/infix-services@2024-05-30.yang"
 	"ieee802-ethernet-interface@2019-06-21.yang"
-	"infix-ethernet-interface@2024-02-27.yang"
-	"infix-factory-default@2023-06-28.yang"
-	"infix-interfaces@2024-09-23.yang -e vlan-filtering"
+	"infix/infix-ethernet-interface@2024-02-27.yang"
+	"infix/infix-factory-default@2023-06-28.yang"
+	"infix/infix-interfaces@2024-09-23.yang -e vlan-filtering"
```

Comment:
Why the extra sub-directory?  I feel this would make it even more cumbersome to work with `yanglint` et al as a developer.  I think I'd prefer to keep them in the same directory as their "parent" IETF/IEEE model.

Reply:
Because it got really messy when all infix modules had two entries, one with date and one without. Maybe have the symlink in the same  directory as the ietf models, infix-system@2038-01-19.yand => infix/infix-system.yang?

Reply:
Ah, of course, didn't consider that, thanks for clarifying.  But I just check with @wkz and he agrees that we don't need the subdirectory.

Reply:
I remove it, no problem.